.circleci/config.yml

```diff
@@ -32,7 +32,6 @@ jobs:
             - .eslintcache
             - ~/.electron
             - ~/Library/Caches/Yarn/v1
```

Comment:
I just had a this path on my new machine and I only have a `v4` key. Maybe this needs to be updated?

```
$ ls ~/Library/Caches/Yarn/
v4
```

Reply:
yea, I think you're right since I got the same result.